server/game/cards/12-KotI/GoldenStorm.js

```diff
@@ -0,0 +1,43 @@
+const DrawCard = require('../../drawcard.js');
+
+class GoldenStorm extends DrawCard {
+    setupCardAbilities(ability) {
+        this.reaction({
+            when: {
+                afterChallenge: event => (
+                    event.challenge.winner === this.controller &&
+                    event.challenge.challengeType === 'military' && 
+                    event.challenge.attackingPlayer === this.controller                  
+                )
+            },
+            target: {
+                // activePromptTitle: 'Select a character',  // <- default prompt message, overridable
```

Comment:
Let's remove this comment, since it's talking about how to use the target API and not anything about this specific card.